Library/Formula/psqlodbc.rb

```diff
@@ -1,9 +1,24 @@
-require "formula"
-
 class Psqlodbc < Formula
-  homepage "http://www.postgresql.org/"
-  url "http://ftp.postgresql.org/pub/odbc/versions/src/psqlodbc-09.03.0300.tar.gz"
-  sha1 "0f41b4678b513aa70f14b03803af92a7abf1e179"
+  homepage "https://odbc.postgresql.org"
+
+  stable do
+    url "http://ftp.postgresql.org/pub/odbc/versions/src/psqlodbc-09.03.0400.tar.gz"
```

Comment:
Can you switch this `url` to https, please?:
https://ftp.postgresql.org/pub/odbc/versions/src/psqlodbc-09.03.0400.tar.gz